lib/routes/rulesController.js

```diff
@@ -29,7 +29,10 @@ var rules = require('../models/rules'),
     myutils = require('../myutils');
 
 function GetAllRules(req, resp) {
-    rules.FindAll(req.service, req.subservice, function(err, data) {
+    var limit = req.query.limit ? req.query.limit : 1000;
```

Comment:
Are they new parameters for the plain rules API? In that case, they should documented.

Reply:
New section about add documentation was added to this issue

Reply:
fixed in 13c5b6d687194887fbc85ae977d6589c90ef170f